docs/src/main/asciidoc/getting-started-dev-services.adoc

```diff
@@ -0,0 +1,353 @@
+
+Reading and writing to the database seems to be working well, but that's a bit unexpected.
+Where did a PostgreSQL database come from? You didn't set anything up.
+
+The database is being managed using xref:{docfile}/dev-services.adoc[dev services].
+Dev services take care of stopping and starting services needed by your application.
+Because you
+included the `postgresql-jdbc` dependency, the database is a containerised PostGresQL database.
+If you'd added `postgresql-mysql` insead, you would have gotten a containerised MySQL database.
+
+If you like, use your container tool to see what containers are running.
+For example, if you're using Docker, run `docker ps`, and for podman, run `podman ps`.
+You'll see something like the following:
+
+----
+ff88dcedd899  docker.io/library/postgres:14  postgres -c fsync...  20 minutes ago  Up 20 minutes  0.0.0.0:34789->5432/tcp  nostalgic_bassi
+----
+
+Stop Quarkus and run `docker ps` again.
+You should see nothing running (it may take a few moments for containers to shut down).
+Quarkus will automatically stop the container when your application stops.
+
+=== Initialising services
+
+If you play with your code some more, you may notice that sometimes, after making an application change, http://localhost/hello/names doesn't list any names.
+What's going on? By default, in dev mode, with a dev services database,
+ Quarkus configures Hibernate database generation to be `drop-and-create`.
+ See the xref:{docfile}/hibernate-orm.adoc#quarkus-hibernate-orm_quarkus.hibernate-orm.database-database-related-configuration[Hibernate configuration reference] for more details.
+ If a code change triggers an application restart, the database tables
+ will be dropped (deleted) and then re-created.
+
+This is convenient, but what if you'd prefer the database to always have content?
+That would make testing easier.
+If you provide an `import.sql` file, Quarkus will use that to initialise
+the database on each start.
+
+1. Make a `src/main/resources/import.sql` file in your project
+2. Add the following SQL statements:
+
+[source, sql]
+----
+INSERT INTO Greeting(id, name)
+VALUES (nextval('Greeting_SEQ'), 'Alice');
+INSERT INTO Greeting(id, name)
+VALUES (nextval('Greeting_SEQ'), 'Bob');
+----
+
+Now, hit `s` in your dev mode session, to force a full restart. Then visit http://localhost:8080/hello/names.
+
+You'll see that Alice and Bob are always included in the list of names.
+
+== Controlling dev services
+
+=== Using an external database instead
+
+What if you'd rather use an external database that you manage yourself?
+Add the following to `src/main/resources/application.properties`:
+
+[source, properties]
+----
+# configure your datasource
+quarkus.datasource.db-kind = postgresql
+quarkus.datasource.username = leopold
+quarkus.datasource.password = bloom
+quarkus.datasource.jdbc.url = jdbc:postgresql://localhost:5432/mydatabase
+----
+
+This tells Quarkus that you don't want it to start a dev service,
```

Comment:
```suggestion
This tells Quarkus that you don't want it to start a Dev Service,
```

Reply:
I do wish I'd got the capitalisation of Dev Service right/the way we want it to be, in an article that's _all about dev services_. :)